src/App.jsx

```diff
@@ -1,17 +1,14 @@
-import './css/App.css';
-import { api } from './api';
-import { Heading, SimpleGrid } from '@chakra-ui/react';
+import "./css/App.css";
+import { Container } from "@chakra-ui/react";
+import { Header } from "./components/Header";
+import { Footer } from "./components/Footer";
 
 function App() {
-
-  api.get('/random').then(response => console.log(response.data.value));
-
   return (
-    <SimpleGrid>
-      <Heading>
-        Ahoj
-      </Heading>
-    </SimpleGrid>
+    <Container>
+      <Header />
+      <Footer />
+    </Container>
```

Comment:
It would be better to create Layout component, where you solve layout (Header, Footer, Main content) and the page itself will be passed via children props.